smart-events-user-dashboard/src/Login.tsx

```diff
@@ -1,34 +1,54 @@
 import * as React from 'react';
+import './App.css'
 import {
-  Container, 
+  Box,
   Avatar, 
   TextField, 
-  Button 
+  Button,
+  Card,
+  CardMedia, 
+  Typography,
 } from '@mui/material';
 import '@fontsource/roboto/300.css';
 import '@fontsource/roboto/400.css';
 import '@fontsource/roboto/500.css';
 import '@fontsource/roboto/700.css';
+import { 
+  flexbox,
+  spacing,
+} from '@mui/system';
 
 function Login() {
-
   return (
-    <div className="Login">
-      <Container>
-        <Avatar 
-          alt="Yellow Jacket Logo"
-          src="/src/assets/yellow-jacket-logo.png"
-          variant="square"
+    <Card
+      sx={{
+        p: 5,
+        height: 350,
+        display: 'flex',
+        flexDirection: 'column',
+        flexwrap: 'nowrap',
+        justifyContent: 'space-between',
+        alignItems: 'center',
+      }}>
+      <Box sx={{display: 'flex', flexDirection: 'column', alignItems: 'center'}}>
+        <CardMedia
+          component='img'
+          image='/src/assets/yellow-jacket-logo.png'
+          alt='Yellow Jacket Logo'
           sx={{
```

Comment:
TL;DR. Perfect for a demo. Great job Kobe. We'll _gradually_ change over when this goes into prod. For prod work, we design for the smallest screen first, and prefer to work with percentages. These sorts of "relative" units are perfect when paired with [FlexBox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/). 

TL:
Traditionally, we don't touch the "size" (`sx`) operator on MUI. This is because we prefer the design to "flow" rather than to be constrained artificially. Another thing about the `sx` operator is that it overriding the current theme. If we want to be able to globally style our app, any use of the `sx` operator will override these styles (from my understanding, please tell me if I am wrong. You have used this feature more than I have).

Imagine it this way:
There are two ways of designing. Designing for **your** screen or designing for **all** screens. Whenever you use `sx` you are likely designing for the vast majority of screens, using **your** screen as the reference point. Since we will be likely demoing using your screen or a screen of your size, this does not matter.

Outside of demo material, we design using a "smallest screen first" principle, and we don't design using size constraints. Note that this is a VERY VERY lofty goal, and we never 100% hit it.

With this "smallest screen first" principle, we still worry about what it will eventually look like on a large screen. So, instead of using px values (what `sx` uses), we use percentages. In a three column layout

A A A

we say that each column is 33% of the screen, rather than saying each column is (1920 / 3)px. This ensures that our design works on a large screen (4k) and a small screen (1k).

Note that this is NOT bad practice for an MVP. You did great work here.

I know you love to learn, so please recognize this is a healthy criticism rather than a scolding remark. I love the work you've done here, and I think it will vastly push forward our progress towards a demo-able MVP. Great job Kobe. Keep striving for excellence.

Reply:
Thank you for your feedback, I appreciate you taking your time to review my work and guide me toward best practice for MUI. Your distinction between demos/MVPs and the ideal deployable solution helps me understand when to use absolute vs. relative sizing values, and your comments on the `sx` operator help me see where it is appropriate to use in our code.